.github/workflows/docker-release-workflow.yaml

```diff
@@ -32,7 +30,7 @@ jobs:
       - name: Login to Docker Hub
         uses: docker/login-action@v3
         with:
-          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          username: 'concordium'
```

Comment:
Won't this break the flow? It has to match the username used in the different repositories. I'm about to add a workflow from the node repository for the collector-backend where it's not going to work.

Reply:
Then we update the password and we are fine